internal/driver/cli.go

```diff
@@ -58,6 +59,7 @@ func parseFlags(o *plugin.Options) (*source, []string, error) {
 	flagTools := flag.String("tools", os.Getenv("PPROF_TOOLS"), "Path for object tool pathnames")
 
 	flagTimeout := flag.Int("timeout", -1, "Timeout in seconds for fetching a profile")
+	flagHTTPPort := flag.Int("http", 0, "Present interactive web based UI at the specified http port")
```

Comment:
Would it be possible to make the "http" another command here - https://github.com/google/pprof/blob/1b8137e88907127c25b226f856452f42d08ec336/internal/driver/commands.go#L87?

I felt this would integrate more naturally, but maybe I am missing something.